### ISSUE TITLE
[tomirons] Forum installation command

### ISSUE THREAD

**Issue by AndreasElia** [AndreasElia/laravel-forum#26]
The `forum:install` command requires the addition of the ability to publish the policy stubs to the main laravel directory of which the package gets installed into.

For example, the following:
```
Console/Stubs/DiscussionPolicy.stub
```

Will be published as:
```
App/Policies/DiscussionPolicy.php
```

**Comment by ConnorVG** [comment]
I feel like `Console/Stubs/DiscussionPolicy.stub` should be `Console/stubs/discussionPolicy.stub` or `discussion-policy.stub` – it's not a class, it's technically data

**Comment by AndreasElia** [comment]
I like the idea of `Console/stubs/discussionPolicy.stub` - thanks!

**Comment by tomirons** [comment]
I'll work on this.

**Comment by AndreasElia** [comment]
Fixed in #46
